benchmark/bench_ondemand.cpp

```diff
@@ -8,19 +8,22 @@ SIMDJSON_PUSH_DISABLE_ALL_WARNINGS
 SIMDJSON_POP_DISABLE_WARNINGS
 
 #include "partial_tweets/ondemand.h"
-#include "partial_tweets/iter.h"
+// #include "partial_tweets/iter.h"
```

Comment:
We probably want to clean out dead comments in time.

Reply:
Oh yeah! I wanted them there to make it clear I wasn't maintaining the iterator versions, but I'll remove these comments before we merge.

I changed how iteration worked; I actually think it'll be worth restoring raw iteration, but my priority here was a safe `value_iterator` that tracked its own depth.